bin/motion-send.sh

```diff
@@ -9,6 +9,12 @@ source $DIR/../telegram.conf
 ## Get the name of the camera which detected a motion (done by parameter %$ in motioneye webinterface command)
 var_cameraName=$1
 
+## Get the id of the camera which detected a motion (done by parameter %t in motioneye webinterface command)
+var_cameraId=$2
+
+var_mediaDir="$(dirname $(grep target_dir /etc/motioneye/camera-$var_cameraId.conf | cut -d ' ' -f 2))"
+var_mediaCameraDir="$var_mediaDir/$var_cameraName"
```

Comment:
Do we really need to create that variable? Couldn't we just join $var_mediaDir/$var_cameraId on line 58/59?

Reply:
```suggestion
```